api/app/controllers/spree/api/images_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'private_address_check'
```

Comment:
We should either add this to the gemspec _or_ only require it when the attachment is a URI, making the functionality and dependency optional.

Reply:
Yeah, I felt like I might be doing this wrong.  Any advice on a proper implementation is welcomed.  For now I'll do your latter suggestion.